src/configuration/services/mysql.md

```diff
@@ -98,17 +98,17 @@ The easiest way to download all data in a MariaDB instance is with the Platform.
 platform db:dump
 ```
 
-If you have multiple SQL databases it will prompt you which one to export. You can also specify one by relationship name explicitly:
+By default the file will be uncompressed. If you want to compress it, use the `--gzip` (`-z`) option:
 
 ```bash
-platform db:dump --relationship database
+platform db:dump --gzip
```

Comment:
Do we only support gzip?  bzip2 (as the previous example used) has much better compression for things like giant text files.